packages/iran/README.md

```diff
@@ -0,0 +1,49 @@
+# 🇮🇷 Iran
+
+Contains all the states and the Federal District of Iran:
+
+- Alborz
+- Ardebil
+- Bushehr
+- Chahar Mahall and Bakhtiari
+- East Azarbaijan
+- Esfahan
+- Fars
+- Gilan
+- Golestan
+- Hormozgan
+- Hamadan
+- Ilam
+- Kohgiluyeh and Buyer Ahmad
+- Khuzestan
+- Kermanshah
+- Kordestan
+- Kerman
+- lorestan
+- Mazandaran
+- Markazi
+- North Khorasan
+- Qom
+- Qazvin
+- Sistan and Baluchestan
+- South Khorasan
+- Semnan
+- Tehran
+- West Azarbaijan
+- Yazd
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/iran`
+
+### yarn
+
+`yarn add @svg-maps/iran`
+
+## Credits
+
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/iran)
```

Comment:
Could you complete the documentation please? You can look at the [contributing guide](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md) or at another map (e.g. [Canada](https://github.com/VictorCazanave/svg-maps/tree/master/packages/canada))